bundle.yaml

```diff
@@ -46,6 +61,12 @@ relations:
   - ["finos-legend-sdlc-k8s", "finos-legend-gitlab-integrator-k8s"]
   - ["finos-legend-engine-k8s", "finos-legend-gitlab-integrator-k8s"]
   - ["finos-legend-studio-k8s", "finos-legend-gitlab-integrator-k8s"]
+  # Gitlab relations:
+  - ["finos-legend-gitlab-integrator-k8s", "gitlab-k8s"]
```

Comment:
What is gitlab-k8s?

Reply:
Was working on something in parallel, I forgot to remove this one. Good catch. :)